test/server.test.js

```diff
@@ -1,14 +1,536 @@
 const chai = require('chai');
 const chaiHttp = require('chai-http');
+const jwt = require('jsonwebtoken');
 const server = require('../app/server'); // Adjust the path as needed
+require('dotenv').config();
+
+const { SECRET_KEY } = process.env;
+const { validateRequestToken } = require('../app/server');
+const { users } = require('../app/server');
 
 const should = chai.should();
 chai.use(chaiHttp);
 
-// TODO: Write tests for the server
+// create constants for tests
+const validUser = { userName: 'yellowleopard753', password: 'jonjon' };
+const user = users.find((u) => u.login.username === validUser.userName);
+const invalidUsers = [
```

Comment:
always include the word mock as these are not real
```suggestion
const invalidUsersMock = [
```